docusaurus/src/theme/SearchBar/DocSearchModal.js

```diff
@@ -0,0 +1,214 @@
+import React, { useMemo } from 'react';
+import algoliasearch from 'algoliasearch/lite';
+import { createAutocomplete } from '@algolia/autocomplete-core';
+import url from 'url';
+import { useLocation } from '@docusaurus/router';
+import { Footer } from './Footer';
+import { SearchBox } from './SearchBox';
+import { Hits } from './Hits';
+
+import { useTouchEvents } from './useTouchEvents';
+import { useTrapFocus } from './useTrapFocus';
+
+import { platformMapping, folderMapping } from '../../../constants';
+
+// It doesnt make sense to use env vars for this while we are still
+// keeping the zipped file on the repo
+const algoliaClient = algoliasearch(
```

Comment:
This is a read-only api key. I think it doesnt make sense to add it to environment variables right now once we will end up exposing it on the gzip file anyways.